test/pycheck/motherduck_test.py

```diff
@@ -56,6 +56,37 @@ def test_md_create_table(md_cur: Cursor, ddb):
         md_cur.sql("CREATE TABLE t4(a int) USING heap")
 
 
+def test_md_read_scaling(pg: Cursor, ddb, default_db_name, md_test_user):
+    weird_db_name = "some 19 really $  - @ weird name 😀 84"
+    try:
+        ddb.sql(f'DROP DATABASE IF EXISTS "{weird_db_name}";')
+    except Exception:
+        pass
+
+    ddb.sql(f'CREATE DATABASE "{weird_db_name}";')
+    pg.search_path = f"ddb${default_db_name}, public"
+    with pg.cur() as cur:
+        hint = "abc123"
+        cur.sql(f"SET duckdb.motherduck_session_hint = '{hint}';")
+
+        # Sanity check
+        assert cur.sql("SHOW duckdb.motherduck_session_hint;") == hint
+
+        cur.execute(
+            f"SELECT duckdb.enable_motherduck('{md_test_user['token']}', '{weird_db_name}')"
```

Comment:
Does this actually do something? MotherDuck is already enabled right? We should at least check what the default database is.

Reply:
Though agree a test on the default DB would be even better. Should be trivial with `SHOW` right?